regression-tests/25-compile-c64/Makefile

```diff
@@ -0,0 +1,14 @@
+EXAMPLESDIR=../../examples
+TOOLSDIR=../../tools
+
+# build c64 tests. Many examples won't fit in memory or won't compile due to the
+# lack of int64_t in cc65, so only a subset of IPv4 is tested.
+EXAMPLES = irc/c64 \
+telnet-server/c64 \
+webbrowser/c64 \
+webserver/c64 \
+wget/c64
```

Comment:
Right so if they were already being built with no error, how is the change in `uip-nd6.c` needed @tiagoshibata ?